src/core2/tutorials/howtostart/run-your-first-program.md

```diff
@@ -39,18 +41,27 @@ Click "Add new device".
 
 ![image](/assets/img/howToStart/2_addNewDevice.png)
 
-Enter a name for you CORE2 powered device.
+Enter a name for your CORE2 powered device.
 
 ![image](/assets/img/howToStart/3_enterName.png)
 
-Scan QR code using your hConfig app.
+Scan QR code using the hConfig app.
 
 ![image](/assets/img/howToStart/4_scanQr.png)
 
 Well done! You just added your first device to the cloud!
 
 ![image](/assets/img/howToStart/5_devAdded.png)
 
+### Connecting via the command line ###
+
+If you have CORE2-ROS, you can also connect to the cloud via the command line. This is recommended only if you already know how to open terminal or connect via SSH to your device.
```

Comment:
"This is recommended only if you are able to open terminal or (...)" - would be more polite (doesn't refer to user knowledge)